<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="q1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Add up all the even elements of the fibonacci sequence below 1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_q1_answer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="q1_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Write a function that will calculate the area of a circle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_q2_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_q2_inputl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>r:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_q2_answer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3.Write a recursive function that calculates the fibonacci sequence up to 1000</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_q3_answer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4. Parse xml</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_q4_inputfile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_q4_outputfile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_q2_inputl_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input folder path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_q2_inputl_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output folder path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_parse">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
